usagov/vote-gov-drupal
# VOTE-1178: replace uglify dependency

## Jira ticket (required)
[Insert Jira ticket link](https://bixal-projects.atlassian.net/browse/VOTE-1178)

## Description (optional)
Replace uglify-es with superseder uglify-js

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Testing steps
1. Inside theme directory run "npm run build"
2. Confirm no errors and dist directory builds as expected
3. Confirm site styling looks the same
